packages/hooks/src/use-block-suite-page-references.ts

```diff
@@ -0,0 +1,38 @@
+import type { Page, Workspace } from '@blocksuite/store';
+import { atom, useAtomValue } from 'jotai';
+import { atomFamily } from 'jotai/utils';
+
+import { useBlockSuiteWorkspacePage } from './use-block-suite-workspace-page';
```

Comment:
I just realized that the way we wrote in `useBlockSuiteWorkspacePage` using weakmap can be replaced by `atomFamily` utility

Reply:
yeap

Reply:
I want to make it simple so write it manually